### ISSUE TITLE
/myfilter or similar

### ISSUE THREAD

**Issue by mekb-turtle** [Minehut/Meta#359]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**

In-game
both

**Describe the feature**

Add a command like /myfilter which allows people to hide messages in lobby chat that contain a word or pattern

**How the feature is useful**

People can block bots using words or bypassing the filter without needing a client or a moderator to mute them at the time.

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by ManagePasswords** [comment]
there should be a premade filter system built-in too and mh should remove the antispam if this is added

**Comment by RichieNy** [comment]
I do not see how this would be of any use, the system we currently use has just been integrated with our `/msg`. Having users block their own words seems like it would just add more complications than usefulness. 

Then again, maybe I am missing the point of this suggestion, if so please let me know.

**Comment by mekb-turtle** [comment]
If there was a ton of bots spamming messages with specific words and there wasn't any mods on at the time, people could use the command to block messages containing those words.

**Comment by RichieNy** [comment]
Though that could be a good idea, I have a feeling that this wouldn't be possible due to the current system we are using. If you are being spammed by bots, the player could just turn their Direct Messages to "Friends Only". Of course, this is just my opinion with this suggestion.

**Comment by LukasmanMHdude** [comment]
for some reason I feel like this is a ploy to add a /ad bypass...

**Comment by Sniped** [comment]
Yeah as Richie said, I feel like this is unnecessary even if it's possible in CommunitySift. If you have a suggestion for how our global filter can be updated, feel free to make it on our [discord server](https://discord.gg/pgXMSFh). Otherwise, I don't believe that users should be making their own sub filters when it can just be implemented into the global filter.

**Comment by ManagePasswords** [comment]
Hypixel has it too, it is very useful in some cases in my opinion.

**Comment by mekb-turtle** [comment]
> Hypixel has it too, it is very useful in some cases in my opinion.

Totally where I didn't get the idea from. But that server rarely has bots.

Anyways I'm mostly focusing on bots in lobby chat when a mod isn't there than PMs. PMs can be disabled but if you have someone who you're genuinely messaging it wouldn't be good to just turn off PMs.
